File: typescript/imzx/python/hello.py
# Minimal Hello World example
from claude_agent_sdk import ClaudeAgent

# Create a minimal agent instance
agent = ClaudeAgent(
    name="HelloWorldAgent",
    system_prompt="You are a friendly assistant.",
)

# Simple query example
def main():
    try:
        response = agent.query("Hello! What is your name?")
        print(f"Response: {response.content}")
    except Exception as e:
        print(f"Error: {e}")

if __name__ == "__main__":
    main()

File: typescript/imzx/typescript/src/hello.ts
// Minimal Hello World example
import { ClaudeAgent } from '@anthropic-ai/claude-agent-sdk';

// Create a minimal agent instance
const agent = new ClaudeAgent({
  name: 'HelloWorldAgent',
  systemPrompt: 'You are a friendly assistant.',
});

// Simple query example
async function main() {
  try {
    const response = await agent.query('Hello! What is your name?');
    console.log('Response:', response.content);
  } catch (error) {
    console.error('Error:', error);
  }
}

main();
